settlement/dymension/dymension.go

```diff
@@ -407,8 +407,14 @@ func (d *HubClient) convertBatchToMsgUpdateState(batch *types.Batch, daClient da
 		return nil, err
 	}
 
-	DAMetaData := &settlement.DAMetaData{
-		Height: daResult.DAHeight,
+	height := uint64(0)
+	if daResult.MetaData != nil {
+		height = daResult.MetaData.Height
+	}
+
+	//TODO (srene): Include extra dataMetaDATA to the Hub for inclusion validation
+	DAMetaData := &da.DAMetaData{
+		Height: height,
```

Comment:
Why are we jus not setting the height to the daResult.Height right away like before?

Reply:
this was to avoid null pointer errors caused by a test returning empty batch result, which should not happen anyway because it will panic before. I adapted the test and removed this